lib/perl/Genome/Model/ReferenceAlignment/Command/LibraryDuplicationRates.pm

```diff
@@ -0,0 +1,63 @@
+package Genome::Model::ReferenceAlignment::Command::LibraryDuplicationRates;
+
+use strict;
+use warnings;
+
+class Genome::Model::ReferenceAlignment::Command::LibraryDuplicationRates {
+    is => 'Command::V2',
+    has => [
+        builds => {
+            is_many => 1,
+            is => 'Genome::Model::Build::ReferenceAlignment',
+            shell_args_position => 1,
+            doc => 'List of builds to report on.',
+        },
+    ],
+};
+
+sub help_brief {
+    'A command to print Picard MarkDuplicates duplication metrics for a reference alignment build.'
+}
+
+sub help_synopsis {
+    'Dump reference alignment build duplication metrics to standard out in tab-delimited format.'
+}
+
+sub help_detail{
+    return <<"EOS"
+All reference alignment builds currently use Picard MarkDuplicates to flag PCR duplicates after merging the individual instrument data alignments.
+One output from Picard MarkDuplicates is a metrics file that provides a per library breakdown of PCR and optical duplicate reads.  This tool simply parses the metrics file and dumps those metrics along with basic build/model information in tab-delimited form.
+EOS
+}
+
+sub execute {
+    my $self = shift;
+
+    my $writer;
+    for my $build ($self->builds) {
+        my $library_metrics = $build->mark_duplicates_library_metrics_hash_ref;
```

Comment:
Just an FYI - Calling this function causes the metrics to be set on the build unless they were set previously. It also does not check the metrics first, where it could return the values in the DB. If the mark dup files do not exist, the values could still be in the database, but would not be returned from this function.